core/controllers/community_dashboard.py

```diff
@@ -204,3 +205,29 @@ def get(self):
         }
 
         self.render_json(self.values)
+
+
+class UserCommunityRightsDataHandler(base.BaseHandler):
+    """Provides lessons content which can be translated in a given language."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self):
+        """Handles GET requests."""
+        if self.username:
+            community_rights = user_services.get_user_community_rights(
+                self.user_id)
+            self.render_json({
+                'can_review_translation_in_languages': (
+                    community_rights.can_review_translation_in_languages
+                    if community_rights else []),
+                'can_review_voiceover_in_languages': (
+                    community_rights.can_review_voiceover_in_languages
+                    if community_rights else []),
+                'can_review_questions': (
+                    community_rights.can_review_questions
+                    if community_rights else False)
+            })
+        else:
+            self.render_json({})
```

Comment:
Perhaps better to use [], [], False and return a dict in the same format?

Unless this handler should not be called for guests, in which case this should raise an error.